virt/virtual_machines/virt-manage-vmis.adoc

```diff
@@ -6,11 +6,18 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-If you have standalone virtual machine instances (VMIs) that were created independently outside of the {VirtProductName} environment, you can manage them by using the web console or the command-line interface (CLI).
+If you have standalone virtual machine instances (VMIs) that were created independently outside of the {VirtProductName} environment, you can manage them by using the web console or by using `oc` or xref:../../virt/virt-using-the-cli-tools.html#virt-using-the-cli-tools[`virtctl`] commands from the command-line interface (CLI).
+
+The `virtctl` command provides more options than the `oc` command, for example, pausing, unpausing, or exposing a port.
```

Comment:
The reasoning for this suggestion is similar to my previous comment; I'm just less worried about the maintenance aspect when comparing `oc` to `virtctl`. For both sets of examples, I removed one of the options, but feel free to re-add if necessary.

```suggestion
The `virtctl` command provides more virtualization options than the `oc` command. For example, you can use `virtctl` to pause a VM or expose a port.
```

Reply:
I'm OK with the suggestion to add "virtualization" to options. It seems a little hair-splitty to me but clarity is always best!